ecpy_hqc_legacy/tasks/tasks/instr/views/spdev_views.enaml

```diff
@@ -69,6 +69,13 @@ enamldef DemodSpView(InstrView): view:
                     "then average.\nThis allows to use a ref Iref, Qref "
                     "and correct each I,Q pair by the ref, then average.")
 
+    Label: num_loop:
+        text = 'Number of pulse sequences in loop'
```

Comment:
The impact of this parameter needs to be detailed probably in the tooltip of the field

Reply:
added a tooltip. Let me know if it is clear